templates/cartridge/Dockerfile.build.cartridge

```diff
@@ -0,0 +1,14 @@
+# Simple Dockerfile
+# Used by `pack` command as a base for build image
+# when --use-dcoker option is specified
+
+# The base image must be centos:8
+FROM centos:8
+
+# Here you can install some packages required
+#   for your application build
+
+# RUN set -x \
+#    && curl -sL https://rpm.nodesource.com/setup_8.x | bash - \
```

Comment:
Node 8 is no longer supported (https://blog.risingstack.com/update-nodejs-8-end-of-life-no-support/)

Reply:
10.x